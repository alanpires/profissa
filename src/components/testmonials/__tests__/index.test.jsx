import React from "react";
import Testmonials from "../index";
import renderer from "react-test-renderer";

describe("Testmonials Test", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(
        <Testmonials image={""} name={"Roberto"} feedback={"muito Bom"} />
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });

  it("renders correctly", () => {
    const tree = renderer
      .create(
        <Testmonials
          image={"Com imagem"}
          name={"Fernando"}
          feedback={"Muito Bom"}
        />
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
