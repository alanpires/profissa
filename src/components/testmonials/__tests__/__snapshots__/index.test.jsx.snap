// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testmonials Test renders correctly 1`] = `
<div
  className="sc-bdnylx jUGZkC"
>
  <div
    className="sc-gtssRu eEZLDO"
  >
    <div
      className="sc-dlnjPT hDdslF"
    >
      <img
        className="sc-hKFyIo fJAUKr"
        src=""
      />
    </div>
    <p>
      Roberto
    </p>
  </div>
  <span>
    muito Bom
  </span>
</div>
`;

exports[`Testmonials Test renders correctly 2`] = `
<div
  className="sc-bdnylx jUGZkC"
>
  <div
    className="sc-gtssRu eEZLDO"
  >
    <div
      className="sc-dlnjPT hDdslF"
    >
      <img
        className="sc-hKFyIo fJAUKr"
        src="Com imagem"
      />
    </div>
    <p>
      Fernando
    </p>
  </div>
  <span>
    Muito Bom
  </span>
</div>
`;
